nvm.sh

```diff
@@ -2419,13 +2421,51 @@ nvm() {
             PROVIDED_REINSTALL_PACKAGES_FROM="$(nvm_echo "$1" | command cut -c 22-)"
             REINSTALL_PACKAGES_FROM="$(nvm_version "$PROVIDED_REINSTALL_PACKAGES_FROM")" ||:
           ;;
+          --skip-default-packages)
+            SKIP_DEFAULT_PACKAGES=true ||:
+          ;;
           *)
             ADDITIONAL_PARAMETERS="$ADDITIONAL_PARAMETERS $1"
           ;;
         esac
         shift
       done
 
+      if [ -z "$SKIP_DEFAULT_PACKAGES" ] && [ -f "${NVM_DIR}/default-packages" ]; then
+        local DEFAULT_PACKAGES=""
```

Comment:
please always declare `local` vars in a separate line from initializing them; this line as-is won't work in `ksh` which doesn't support `local`.